leda/components/Validation/predefinedValidators.ts

```diff
@@ -24,7 +24,7 @@ export const PREDEFINED_VALIDATORS: Record<PredefinedValidator, NormalizedValida
   },
   password: {
     validator: validate.password,
-    invalidMessage: 'Пароль должен содержать не менее 8-и символов, латинские строчные, прописные буквы и цифры',
+    invalidMessage: 'Use at least 8 latin lower- and uppercase letters and numbers',
```

Comment:
```suggestion
    invalidMessage: 'Use at least 8 latin lowercase and uppercase letters and numbers',
```
Looks better imo

Reply:
Do you think it's worth adding special characters as well?

Reply:
maybe, as a new feature, you can add it to the board
just numbers look lonely